### DISCUSSION TITLE
`graph --deploy` should show most recently deployed version

### DISCUSSION THREAD

**Discussion by marcusrein** [graphprotocol/graph-tooling#1723] [Ideas]
Doing `graph --deploy` should let me know what the most recently deployed version is right from the CLI. Currently, I have to double check which version I have deployed in Studio to make sure my versioning is correct for every update. It would be nice if it would automatically populate that data right in the CLI when I use `graph --deploy`

EXAMPLE:

```
graph deploy --studio iotex-demo-subgraph
Currently deployed version: 0.0.3
Which new version label to use? (e.g. "v0.0.4"):
```

@saihaj and I chatted and he added this context:

"We have chatted about this in past. The reasons are:
1. Lack for the availability of the data. Studio would need to expose some endpoints for this.
2. in a multi studio world how does it work? Do all providers have to expose some endpoint ?
3. There’s not real strictness that versions follow. We say it should be semver but not enforced.

Ideally some sort of registry with a public API solves many  of these issues."

**Comment by YaroShkvorets** [comment]
Tracked here: https://github.com/graphprotocol/graph-tooling/issues/1791
